wire-ios-data-model/Source/Model/FeatureConfig/FeatureService.swift

```diff
@@ -230,6 +230,29 @@ public class FeatureService {
         }
     }
 
+    // MARK: - MLS Migration
+
+    public func fetchMLSMigration() -> Feature.MLSMigration {
+        guard
+            let feature = Feature.fetch(name: .mlsMigration, context: context),
+            let featureConfig = feature.config
+        else {
+            return .init()
+        }
+
+        let config = try! JSONDecoder().decode(Feature.MLSMigration.Config.self, from: featureConfig)
```

Comment:
is it safe to use `try!` here?

Reply:
It shouldn't break but I guess it is possible. I wrote it like this to be consistent with the rest of the class, but I agree we should handle this better.

What do you think if we return the default config in case of a decoding error?

Reply:
I removed the force try, returning the default config and logging the error in case it fails.